modules/meeting/app/controllers/meetings_controller.rb

```diff
@@ -315,6 +316,7 @@ def convert_params
     # Force defaults on participants
     @converted_params[:participants_attributes] ||= {}
     @converted_params[:participants_attributes].each { |p| p.reverse_merge! attended: false, invited: false }
+    @converted_params[:send_notifications] = params[:send_notifications] == '1'
```

Comment:
I believe using the ActiveRecord Boolean caster would be a less error prone approach.
```suggestion
    @converted_params[:send_notifications] = ActiveRecord::Type::Boolean.new.cast params[:send_notifications]
```

Reply:
I tried to use your approach but it doesn't work here [because when the check_box_tag is used, no value is returned on unchecking](https://stackoverflow.com/a/35827427) and `ActiveRecord::Type::Boolean.new.cast` returns nil for that and not false. 

I could have added a `params[:send_notifications] ||= []` as suggested in the SO answer, but the existing approach seems better to me. 

Why would `ActiveRecord::Type::Boolean.new.cast` be a less error prone approach?

Reply:
Got it, it's less error prone, because it can parse a variation of string values to boolean correctly, such as "0", "1", "true", "false". So whenever we decide to use a different tag from `checkbox_tag`, it would still behave the same way. But since it's a bit more complex, then I'm fine with leaving the code as is.